conf/rmwork.conf

```diff
@@ -1,2 +1,18 @@
 RMWORK ?= "0"
 RMWORK[nohash] = True
+
+# By default, remove everything but ${WORKDIR}/tmp aka ${T}. We refuse
+# to remove a directory which is not a subdirectory of ${WORKDIR}.
+RMWORK_DIRS ?= """
+${STAGE_DIR}
+${STAGE_UNPACKDIR}
+${RSTAGE_DIR}
+${FSTAGE_DIR}
+${SRCDIR}
+${S}
+${B}
+${D}
+${PKGD}
+${IMAGE_DIR}
+"""
```

Comment:
Do we really need to list these dirs?  Why not just delete all subdirs of WORKDIR, except tmp?